daft/series.py

```diff
@@ -39,13 +39,31 @@ def from_arrow(array: pa.Array | pa.ChunkedArray, name: str = "arrow_series") ->
             raise TypeError(f"expected either PyArrow Array or Chunked Array, got {type(array)}")
 
     @staticmethod
-    def from_pylist(data: list, name: str = "list_series") -> Series:
+    def from_pylist(data: list, name: str = "list_series", pyobj: str = "allow") -> Series:
```

Comment:
btw you can use: https://docs.python.org/3/library/enum.html#enum.StrEnum for something like this